lib/auto-languageclient.js

```diff
@@ -171,6 +171,18 @@ export default class AutoLanguageClient {
     }
   }
 
+  newEditorSelector(editor: atom$TextEditor, basepath: string): atom$TextEditor => boolean {
```

Comment:
Let's rename this `shouldSyncForEditor` and change the second arg to `projectPath` for consistency.